### ISSUE TITLE
[COMMUNITY FEEDBACK] Mailing list problems

### ISSUE THREAD

**Issue by cjfields** [bioperl/bioperl-live#258]
The Bioperl mailing lists (as well as all other OBF mail lists) have been bown or having serious issues over the last several months due to heavy spam traffic on the OBF mailman server.  I am soliciting community feedback from both developers and users for the next steps forward on resolving this issue (see below).  Please post suggestions or concerns here.

First, [we (OBF) are migrating](https://news.open-bio.org/2017/12/15/mailing-list-consolidation/) to a hosted solution that should provide more stability, with the downside that many lists will be closed and archived due to hosting costs.  The [blog post](https://news.open-bio.org/2017/12/15/mailing-list-consolidation/) details some suggestions.

Currently, we plan on having the main Bioperl list (bioperl-l) remain open; however, there are five others that  will be closed down or merged with the main list:

* Bio-phylo-l – BioPerl Phylogenetics modules (archive)
* BioPerl-announce-l – Bioperl Project Announcements (archive)
* bioperl-dev – BioPerl Developer Core (archive)
* BioPerl-guts-l – BioPerl internals & bug reports (private archive)
* bioperl-microarray – Microarrays in BioPerl (archive)

At the moment I am opting to close and archive all of the above side project lists.  The one with the highest traffic in this group is the 'bioperl-guts-l' list, but there is not really a need for this list anymore since you can opt to follow the commits directly from Github if one chooses.  The others have always been very low traffic and haven't seen any mail (barring the random spam message) in years.

Again, please note your concerns as soon as possible.  We would like feedback by Dec. 28 so that we can decide on the next steps.

Thanks all!

chris

**Comment by hlapp** [comment]
Just for clarification, unless someone can point out a protocol for how to do this safely and with minimal effort, "merging" mailing lists will _not_ include merging the mailing list archives.

As for merging subscribers, it's not clear yet whether that's feasible to do safely (not resulting in double subscriptions etc) and with minimum effort either. It seems a reasonable assumption to make is that the "main" list (bioperl-l) is, at least approximately, a superset of the other lists, in terms of subscribers who remain legit and would actually _want_ to be on the main list. The list for which this may be least or not true is presumably the bioperl-announce-l list.

Hence, for the purpose of this call for feedback, assume that "merging" would not be different from "archiving" in any notable way. If that would prompt strong concerns, please note them here.

**Comment by cjfields** [comment]
@hlapp agreed.  The simplest solution would be simple archival of the side-project lists and migrating bioperl-l (with the current subscribers) to the new hosted instance.

**Comment by cjfields** [comment]
I’m assuming silence is implicit agreement to the proposed changes.  We’ll leave this open for further comments but will proceed with the above plan

**Comment by cjfields** [comment]
Tentative plans are to work on merging mail list subscribers to bioperl-l this coming week, with prior posts to older lists in a read-only archive.

**Comment by cjfields** [comment]
As a note, migration is still planned but the immediate need (namely dealing with the massive spamming issues) has temporarily passed.  We still plan on closing down the above lists and merging them with the main one.
